lib/ldapUser.class.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+include sfConfig::get('sf_root_dir') .'/vendor/password_compat/password.php';
```

Comment:
It looks like autoload will only handle classes, unfortunately. It parses files to determine if they're a class. We could subclass the autoloader class, though, as per:

https://capocasa.net/trait-with-symfony-1-4

I think the thing to do would be if a file, rather than a directory, is specified in autoload.yml then include it without parsing it.